libintl configuration failure

Hi Team,

I have gettext in my machine for other purposes, But I don't need gettext support for my quality build,
But, I could see this error, despite passing "--without-libintl-prefix" in the configure option, is there any reason?

```
./configure \
		--target=powerpc-uclibcspe \
		--host=powerpc-uclibcspe \
		--build=x86_64-linux-gnu \
        	--prefix=/usr \
        	--exec-prefix=/usr \
		--bindir=/usr/bin \
		--sbindir=/usr/sbin \
		--libdir=/usr/lib \
		--libexecdir=/usr/lib \
		--sysconfdir=/etc \
		--datadir=/usr/share \
		--localstatedir=/var \
		--mandir=/usr/share/man \
		--infodir=/usr/share/info \
		--includedir=/usr/include \
		--disable-nls --disable-Werror \
        	--without-libintl-prefix --disable-cracklib-check --enable-pam 

libtool: compile:  uclibcspe-gcc -DHAVE_CONFIG_H -I. -I.. -Os -g -pipe -MT pam_pwquality.lo -MD -MP -MF .deps/pam_pwquality.Tpo -c pam_pwquality.c  -fPIC -DPIC -o .libs/pam_pwquality.o
pam_pwquality.c:18:21: fatal error: libintl.h: No such file or directory
 #include <libintl.h>
                     ^
compilation terminated.
Makefile:627: recipe for target 'pam_pwquality.lo' failed
make[3]: *** [pam_pwquality.lo] Error 1
```

--- a/src/pam_pwquality.c
+++ b/src/pam_pwquality.c
@@ -15,7 +15,6 @@
 #include <ctype.h>
 #include <limits.h>
 #include <syslog.h>
-#include <libintl.h>
 #include <stdio.h>
 #include <pwd.h>
 #include <errno.h>

